apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -532,11 +532,11 @@ export default Vue.extend({
           headerTooltip += ' [Primary Key]'
         }
 
-        let cssClass: string;
+        let cssClass = 'hide-header-menu-icon';
```

Comment:
Going to accept this, but a clean way to do this in future is using an array:

cssClasses = ['x']
cssClasses.join(" ")